admin/includes/template/partials/tplAdminLeadItemRows.php

```diff
@@ -32,4 +32,16 @@
         <?php } ?>
     </tr>
 <?php } ?>
+<?php if ($tplVars['listingBox']['formattedTotals'] != null) { ?>
+<tr>
+<?php $cols = count($tplVars['leadDefinition']['listMap']) + 1; ?>
+<td colspan="<?php echo $cols;?>"><?php zen_draw_separator(); ?></td>

```

Comment:
Okay. Probably best to use CSS for that, such as a corresponding `class` on that cell.

I've never mentioned it yet, but I'd like to retire `zen_draw_separator` because it's using the old-school approach of images for separation and spacers.